commercial/app/AppLoader.scala

```diff
@@ -77,4 +59,15 @@ trait AppComponents extends FrontendComponents with AppLifecycleComponents with
   override lazy val httpErrorHandler: HttpErrorHandler = wire[CorsHttpErrorHandler]
   override lazy val httpFilters: Seq[EssentialFilter] = wire[CommonFilters].filters
   override lazy val httpRequestHandler: HttpRequestHandler = wire[DevParametersHttpRequestHandler]
+
+  lazy val devAssetsController = wire[DevAssetsController]
+  lazy val healthCheck = wire[HealthCheck]
+
+  override lazy val lifecycleComponents = List(
+    wire[LogstashLifecycle],
+    wire[CommercialLifecycle],
+    wire[SwitchboardLifecycle],
+    wire[CloudWatchMetricsLifecycle],
+    wire[CachedHealthCheckLifeCycle]
+  )
```

Comment:
@alexduf Not related to removing `Play.current` but as we mentioned earlier, simplifying AppLoader.
It makes sense to have traits for controllers and services since they are also used in Standalone app but `lifecycleComponents`, `devAssetsController` and `healthCheck` can be defined in `AppComponents`